packages/cryptsetup/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "cryptsetup"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+[package.metadata.build-package]
+releases-url = "https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7"
+
+[lib]
+path = "../packages.rs"
+
+[[package.metadata.build-package.external-files]]
+url = "https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz"
+sha512 = "13eca93cdb00a143d2ca60b6f66ede5adc4072ca0c4bfebd8454a3541e69d269fcdb4afc97ad799e87a999b2bd46c1f31fa924a3d616d72a3337970b1e718d55"
+force-upstream = true
+
+[build-dependencies]
+glibc = { path = "../glibc" }
+device-mapper = { path = "../device-mapper" }
+json-c = { path = "../json-c" }
+libblkid = { path = "../util-linux" }
+libuuid = { path = "../util-linux" }
+libpopt = { path = "../libpopt" }
+systemd = { path = "../systemd" }
```

Comment:
Since systemd also depends on cryptsetup, this would create a dependency cycle, which we'd need to break in one of two ways:
1. adding a `systemd-bootstrap` package or similar (`libsystemd`, ...) with the same sources and a trimmed down build, just enough to bootstrap cryptsetup
2. adding systemd headers and libraries to the SDK - effectively doing the bootstrap there

From a quick glance at the sources, it looks like the dependency is only used to find the path to the systemd tmpfiles.d directory, which we can specify as an argument to the configure script instead.

Reply:
I did remove it completely now as per your recommendation.